akka-projection-cassandra/src/it/scala/akka/projection/cassandra/ContainerSessionProvider.scala

```diff
@@ -33,14 +34,19 @@ final class ContainerSessionProvider extends CqlSessionProvider {
 
 object ContainerSessionProvider {
   private val disabled = java.lang.Boolean.getBoolean("disable-cassandra-testcontainer")
-  private lazy val container = new CassandraContainer()
+
+  @silent private lazy val container: CassandraContainer[_] = {
```

Comment:
I think 2.12 compile will complain about using existential types.

Reply:
It seems to pass the `check` stage.

Reply:
However, I don't mind rolling this back.  It was just part of an earlier experiment and isn't required.

Reply:
It's fine like that. I think I run into it recently, but that was when passing such a type to a method argument. 

Anyway, if it compiles, it's good. 👍